### DISCUSSION TITLE
How to create a transparent image?

### DISCUSSION THREAD

**Discussion by FrankenApps** [GraphiteEditor/Graphite#1216] [Help] [answered]
Hi, I would like to create a PNG image with a transparent background (i.e. without the default white background).
How can I do this? Is it possible at all?

**Comment by Keavon** [comment, marked as answer]
Are you referring to importing or exporting PNG image with a transparent background? For importing, it should respect your transparency. For exporting, that unfortunately isn't possible from the File > Export menu right now, but it should be feasible relatively soon for us to solve that. There's one way you can export with a transparent background, which is clicking one of these buttons in the Properties panel if your layer outputs a bitmap image with transparency. But unfortunately the Frame tool doesn't preserve transparency, which means it can't be used to combine several layers into one. So you'd have to put all your artwork into one layer's node graph if you want to export using these buttons.

![image](https://github.com/GraphiteEditor/Graphite/assets/4388688/80998bf2-93c5-494a-a148-79dc874ff46e)

Note: until we deploy an update (soon) to https://editor.graphite.rs, you'll want to use the latest development release at https://dev.graphite.rs to use that feature.

**Reply by FrankenApps** [reply]
Yeah, I was referring to the File > Export menu and also when creating the canvas I was not able to make the background transparent. Thank you for the detailed information, I will gladly wait for that feature. Keep up the great work 👍

**Reply by 0HyperCube** [reply]
This feature has now been implemented in https://github.com/GraphiteEditor/Graphite/pull/1344 @FrankenApps.
